server/pbenchinacan/container-build.sh

```diff
@@ -44,7 +44,9 @@ fi
 container=$(buildah from ${BASE_IMAGE})
 
 buildah config \
+    --entrypoint /sbin/init \
     --label maintainer="Pbench Maintainers <pbench@googlegroups.com>" \
+    --workingdir / \
```

Comment:
Can we have the working directory be a mount point, like `/srv/pbench/work`?

Reply:
I believe that we can; but, this working directory is principally for `/sbin/init`, not the Pbench server (although, it will presumably be imposed on any other command that one runs in this container)...you still want it to be there?  (I'm not even sure that `init` produces any output or files or anything.)

Reply:
What is the working directory if we don't specify `--workingdir` at all?

Reply:
Apparently the default is the root directory (`/`) unless the (base) image provides some other default.  I would expect that `init` normally runs with `/` as its working directory, but I have no idea why I think that.

Reply:
Per our discussion, I'm going to remove the setting of the work directory and we'll let it default.